### ISSUE TITLE
Include thin-provisioning-tools to included set of packages

### ISSUE THREAD

**Issue by aragilar** [grml/grml#81]
In order to activate lvm volumes which use lvm caching, `/usr/sbin/cache_check` (provided by thin-provisioning-tools) needs to be installed (see https://bugs.debian.org/773731). As grml currently does not include thin-provisioning-tools, volumes which use caching cannot be activated on startup (e.g. via the lvm boot option), not activated manually (naturally one can install thin-provisioning-tools via apt, but that doesn't help with activating on startup).

**Comment by mika** [comment]
Interesting find, thanks for reporting this, we definitely need to solve this for the upcoming stable release. :) Thanks!

**Comment by romanrm** [comment]
Yes please, just had this issue while trying to use grml as my usual rescue system, but this time with some thin volumes. Is there maybe some development snapshot which already has this package included?
